code-scanning/sonarqube.yml

```
@@ -44,23 +44,22 @@ jobs:
       - name: Analyze with SonarQube
 
         # You can pin the exact commit or the version.
-        # uses: SonarSource/sonarqube-scan-action@v1.1.0
-        uses: SonarSource/sonarqube-scan-action@7295e71c9583053f5bf40e9d4068a0c974603ec8
+        # uses: SonarSource/sonarqube-scan-action@v3.1.0
+        uses: SonarSource/sonarqube-scan-action@13990a695682794b53148ff9f6a8b6e22e43955e
```

A new version of the action has just been released:
```suggestion
        # uses: SonarSource/sonarqube-scan-action@v4.0.0
        uses: SonarSource/sonarqube-scan-action@94d4f8ac4aaefccd7fb84bff00b0aeb2d65fcd49
```
@henryju